build.gradle

```diff
@@ -578,10 +578,209 @@ task mapNamedJar(dependsOn: ["mergeV2", mapIntermediaryJar]) {
 	doLast {
 		logger.lifecycle(":mapping minecraft to named")
 
-		mapJar(namedJar, intermediaryJar, mergeV2.output, libraries, "intermediary", "named")
+		mapJar(namedJar, unpickedJar, mergeV2.output, libraries, "intermediary", "named")
 	}
 }
 
+import daomephsta.unpick.api.ConstantUninliner
+import daomephsta.unpick.api.constantmappers.ConstantMappers
+import daomephsta.unpick.api.constantresolvers.ConstantResolvers
+import daomephsta.unpick.api.IClassResolver
+import daomephsta.unpick.constantmappers.datadriven.parser.v2.UnpickV2Reader
+import daomephsta.unpick.constantmappers.datadriven.parser.v2.UnpickV2Writer
+import daomephsta.unpick.constantmappers.datadriven.parser.v2.UnpickV2Remapper
+import daomephsta.unpick.constantmappers.datadriven.parser.MethodKey
+import org.objectweb.asm.ClassReader
+import java.util.jar.JarEntry
+import java.util.jar.JarFile
+import java.util.jar.JarOutputStream
+import org.objectweb.asm.tree.ClassNode
+import org.objectweb.asm.ClassWriter
+import net.fabricmc.mapping.tree.TinyMappingFactory
+
+class JarClassResolver implements IClassResolver, Closeable {
+	private final URLClassLoader classLoader;
+
+	JarClassResolver(Set<File> jars) {
+		this.classLoader = new URLClassLoader(
```

Comment:
Can't we use a java util's jar file instead? jar file afaik supports multi-release jars.